bundles/org.openhab.binding.luftdateninfo/src/main/resources/ESH-INF/i18n/luftdateninfo_de.properties

```diff
@@ -0,0 +1,9 @@
+binding.luftdateninfo.name = LuftdatenInfo Binding
+binding.luftdateninfo.description = Das Binding stellt die Daten der Eigenbau-Sensoren von LuftdatenInfo zur Verf�gung
+
+thing-type.luftdateninfo.particulate.label = Feinstaubsensor
+thing-type.luftdateninfo.particulate.description = Messung der Feinstaub Belastung in der Umgebung 
+thing-type.luftdateninfo.conditions.label = Umgebungssensor
+thing-type.luftdateninfo.conditions.description = Messung der Temperatur, Luftfeuchtigkeit und Luftdruck 
+thing-type.luftdateninfo.noise.label = L�rmsensor
+thing-type.luftdateninfo.noise.description = Messung der Feinstaub Belastung in der Umgebung 
```

Comment:
Copy & paste error

Reply:
OMG -  such a small file, so many errors :( Sorry for that! Will fix it